guides/17_Migration_to_3_0.md

```diff
@@ -61,7 +61,8 @@ await meetingSession.audioVideo.chooseVideoInputDevice(videoInputDeviceInfo.devi
 await meetingSession.audioVideo.startVideoInput(videoInputDeviceInfo.deviceId);
 ```
 
-In v3, you should call `stopVideoInput` to stop the video input stream.
+In v3, you should call `stopVideoInput` to stop the video input stream. `null` is no longer a valid input for 
```

Comment:
Do we need to mention that we remove `null` from type of `Device`?
When I migrating React SDK, several React SDK API use this `Device` type and have to update them accordingly.